akka-typed/src/main/scala/akka/typed/javadsl/Actor.scala

```diff
@@ -58,44 +55,19 @@ object Actor {
    * @see [[Actor#mutable]]
    */
   abstract class MutableBehavior[T] extends ExtensibleBehavior[T] {
-    @throws(classOf[Exception])
-    override final def receiveMessage(ctx: akka.typed.ActorContext[T], msg: T): Behavior[T] =
-      onMessage(msg)
+    private val receive: Receive[T] = createReceive
```

Comment:
could there be any initialization order issues with calling `createReceive` from constructor, e.g. are things in the concrete class intialized first?